linkis-web/src/apps/linkis/module/acrossClusterRule/index.vue

```diff
@@ -0,0 +1,543 @@
+        ],
+        MemoryThreshold: [
+          {required: true, message: this.$t('message.linkis.ipListManagement.notEmpty'), trigger: 'blur'},
+          {validator: this.thresholdValidator, trigger: 'blur'},
+        ],
+        CPUThreshold: [
+          {required: true, message: this.$t('message.linkis.ipListManagement.notEmpty'), trigger: 'blur'},
+          {validator: this.thresholdValidator, trigger: 'blur'},
+        ],
+        MemoryPercentageThreshold: [
+          {required: true, message: this.$t('message.linkis.ipListManagement.notEmpty'), trigger: 'blur'},
+          {validator: this.percentageThresholdValidator, trigger: 'blur'},
+          {type: 'string', max: 4, message: this.$t('message.linkis.ipListManagement.customLen', {length: '4'})}
+        ],
+        CPUPercentageThreshold: [
+          {required: true, message: this.$t('message.linkis.ipListManagement.notEmpty'), trigger: 'blur'},
+          {validator: this.percentageThresholdValidator, trigger: 'blur'},
+          {type: 'string', max: 4, message: this.$t('message.linkis.ipListManagement.customLen', {length: '4'})}
+        ],
+      },
+      // tagIsExist: true,
+      mode: 'create',
+      page: {
+        pageSize: 10,
+        pageNow: 1,
+        totalPage: 0,
+      },
+      userName: '',
+      isRequesting: false,
+      json: '',
+      viewerOptions: {
+        mode: 'view'
+      },
+      editorOptions: {
+        mode: 'code'
+      },
+      showViewModal: false,
+      errorJson: false,
+    }
+  },
+  computed: {
+    mapping () {
+      return (this.modalData.user || 'user') + '-' + (this.modalData.creator || 'creator') + '  -->  ' + (this.modalData.ipList || 'IPList')
+    }
+  },
+  methods: {
+    async getTableData() {
+      try {
+        this.tableLoading = true;
+        let params = {};
+        const keys = Object.keys(this.confirmQuery);
+        for (let i = 0; i< keys.length; i++) {
+          if(this.confirmQuery[keys[i]]) {
+            params[[keys[i]]] = this.confirmQuery[keys[i]];
+          }
+        }
+        params.pageNow = this.page.pageNow;
+        params.pageSize = this.page.pageSize;
+        await api.fetch("/configuration/acrossClusterRule/list", params, "get")
+          .then((res) => {
+            this.datalist = res.acrossClusterRuleList.map((item) => {
+              item.userCreator = item.user + "-" + item.creator;
+              item.parsedRules = JSON.parse(item.rules)
+              // item.createTime = new Date(item.createTime).toLocaleString();
+              return item;
+            })
+            this.page.totalPage = res.totalPage;
+          })
+        this.tableLoading = false;
+      } catch(err) {
+        window.console.log(err);
+        this.tableLoading = false;
+      }
+
+    },
+    async init() {
+      this.loading = true;
+      await this.getTableData();
+      this.loading = false;
```

Comment:
check if `this.loading = false;` will excute  when this.getTableData excute error